src/Compilers/Core/Portable/DiagnosticAnalyzer/SuppressMessageAttributeState.cs

```diff
@@ -230,7 +230,20 @@ private ISymbol SuppressMessageAttribute
             }
         }
 
-        private void DecodeGlobalSuppressMessageAttributes()
+        private ISymbol? UnconditionalSuppressMessageAttribute
+        {
+            get
+            {
```

Comment:
Just checking my understanding: is the purpose of this StrongBox just to allow us to distinguish these cases:
1. _lazySuppressMessageAttribute is not initialized
2. _lazySuppressMessageAttribute is initialized, and its value is null
3. _lazySuppressMessageAttribute is initialized, and its value is non-null

Reply:
Correct, although we don't really need to care whether the value is null or non null, just whether or not it's initialized.